Logger/LogFactory.cs

```diff
@@ -2,10 +2,23 @@
 {
     public class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
+        private string? FileLoggerPath;
+
+        public BaseLogger? CreateLogger(string className)
         {
-            
+            if (FileLoggerPath is string valueOfFileLoggerPath)
```

Comment:
```suggestion
            if (_FileLoggerPath is null)
```
You should just explicitly check for null, not check if the value is a string.